src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers/ClientMethodsAnalyzer.cs

```diff
@@ -23,10 +23,11 @@ public class ClientMethodsAnalyzer : ClientAnalyzerBase
 
         private static void CheckClientMethod(ISymbolAnalysisContext context, IMethodSymbol member)
         {
-            static bool IsCancellationTokenParameter(IParameterSymbol parameterSymbol)
+            static bool SupportsCancellationsParameter(IParameterSymbol parameterSymbol)
             {
-                return parameterSymbol.Name == "cancellationToken" && parameterSymbol.Type.Name == "CancellationToken";
-            }
+                return (parameterSymbol.Name == "cancellationToken" && parameterSymbol.Type.Name == "CancellationToken") ||
+                       (parameterSymbol.Name == "context" && parameterSymbol.Type.Name == "RequestContext");
```

Comment:
Will we add another condition here when we support `RequestOptions` as well?

Reply:
I believe we're also considering a case for HLC convenience methods where RequestOptions can be passed to a method via a property bag.  Is this something we want to lay groundwork for here, or address at the point it is a reality?  (probably the latter)

Reply:
I was thinking about it, but I think it's better to wait till we actually add RequestOptions. Right now it's not clear what the exact design for ROs will be.